src/components/RegionInput.js

```diff
@@ -52,14 +53,20 @@ export const RegionInput = ({
         <Autocomplete
           disablePortal
           freeSolo // Allows custom input outside of the options
-          getOptionLabel={(option) => option.title || option.toString()}
+          getOptionLabel={(option) => option.label || option.toString()}
           value={region}
           inputValue={region}
           data-testid="autocomplete"
           id="regionInput" 
 
-          onInputChange={(event, newInputValue) => {
-            handleRegionChange(newInputValue);
+          onInputChange={(event, newInput) => {
+            let regionValue = newInput;
+            const regionObject = displayRegions.find((option) => option.label === newInput);
+            // If input is selected from one of the options
+            if (regionObject) {
+              regionValue = regionObject.value
+            }
+            handleRegionChange(regionValue, regionToDesc.get(regionValue));
```

Comment:
I feel like the regionToDesc lookup might really want to be in `handleRegionChange`? So it doesn't need a new argument? But we kind of maintain the whole database of regions in RegionInput and it would be odd to do it twice.

I think Shreya is working on the left/right button jump-to-next-region thing in #354 though and that requires having more of an understanding of the region database in the header form, so I think this sort of thing is moving out of RegionInput in the medium term.